src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -142,8 +133,7 @@ size_t ConnectedSubchannel::GetInitialCallSizeEstimate(
 RefCountedPtr<SubchannelCall> SubchannelCall::Create(Args args,
                                                      grpc_error** error) {
   const size_t allocation_size =
-      args.connected_subchannel->GetInitialCallSizeEstimate(
-          args.parent_data_size);
+      args.connected_subchannel->GetInitialCallSizeEstimate();
```

Comment:
I know that the actual parent data size allocation is now done in a different arena allocation; does that affect the arena allocation efficiency or memory usage at all?